app/cardrecord.js

```diff
@@ -103,23 +108,37 @@ class CardRecorder extends MyTrello {
         return deferred.promise;
     }
 
+    checkCommentsForDates(commentList, latest){
+      var myRegex = /(\d\d\/\d\d\/201\d) - \d\d\/\d\d\/201\d/; //Find the first date in the comment string
+      if(!latest){
+        // Reverse order to get first comment
+        commentList = commentList.reverse();
+      }
+      var correctComment = _.find(commentList, function(comment){
+          var match = myRegex.exec(comment.data.text);
+          return match ? match[1] : false;
```

Comment:
For clarity, can this be `return match ? true : false;`?  It took me a minute to grok that `correctComment` is a comment object rather than the value of `match[1]`, and I was confused about why the regex needed to be executed again below.  As-is is completely valid, though - the underscore docs just call for a truthy value, and `match[1]` will be truthy.